src/main/java/org/jenkinsci/test/acceptance/po/WorkflowJob.java

```diff
@@ -37,4 +47,20 @@ public WorkflowJob(Injector injector, URL url, String name) {
     public final Control script = control("/definition/script");
     public final Control sandbox = control("/definition/sandbox");
 
+    public String copyResourceStep(String filePath) {
+        final Resource res = resource(filePath);
+        try (InputStream in = res.asInputStream()) {
+            ByteArrayOutputStream out = new ByteArrayOutputStream();
+
+            try (OutputStream gz = new GZIPOutputStream(out)) {
+                IOUtils.copy(in, gz);
+            }
+
+            return String.format("sh '''base64 --decode << ENDOFFILE | gunzip > %1$s \n%2$s\nENDOFFILE'''\n",
```

Comment:
Yes, the shell command is copied (after removing some parts) but the method signature (and the logic inside it) is different, since `Job#copyResource` returns `void` and makes use of `addShellStep` which is not valid for a `WorkflowJob`.